members/sophiePark.md

```diff
@@ -0,0 +1,15 @@
+# Sophie Se Hyun PARK
+
+## Education
+
+* National University of Singapore, Singapore (B.Comp in Computer Science)
+
+## Work Experience
+
+* [3 months: 2017 May - July] Graphic Designer and Web Developer Intern, Miao Technology Pte Ltd, Singapore
+* [Freelance] Korean-English Translator
+
+## Accomplishments
+
+1. Redesigned UI of existing mobile application on Sketch and Adobe Photoshop CC. Tested UI on target users. Communicated with iOS developers regarding new UI design using Zeplin software. 
```

Comment:
Might want to use bullets instead of numbers to maintain consistency within the document.